pilot/pkg/features/pilot.go

```diff
@@ -446,6 +446,9 @@ var (
 
 	EnableEnvoyFilterMetrics = env.RegisterBoolVar("PILOT_ENVOY_FILTER_STATS", false,
 		"If true, Pilot will collect metrics for envoy filter operations.").Get()
+
+	MulticlusterHeadless = env.RegisterBoolVar("PILOT_MULTICLUSTER_HEADLESS", false,
```

Comment:
I know `PILOT_` is the pattern, but can we drop it?

Reply:
Changed the names